pylint/checkers/__init__.py

```diff
@@ -39,7 +39,8 @@
 15: stdlib
 16: python3
 17: refactoring
-18-50: not yet used: reserved for future internal checkers.
+18: threading
+19-50: not yet used: reserved for future internal checkers.
```

Comment:
```suggestion
18-50: not yet used: reserved for future internal checkers.
```

For now, reverting this might be better. In any case, if we want to change this here it should be `21` as that is the prefix we are now using for this checker.

Reply:
Reverted